### PR TITLE
Slashing Penalty Calculation Change

### PR DESCRIPTION
**Pull request by nisdas**
- [x] Resolves this change in v0.8 https://github.com/ethereum/eth2.0-specs/pull/1217/

### CODE REVIEW

File: shared/params/config.go [hunk 1]
```diff
@@ -57,11 +57,11 @@ type BeaconChainConfig struct {
...
 
 	// Reward and penalty quotients constants.
-	BaseRewardFactor             uint64 `yaml:"BASE_REWARD_FACTOR"`             // BaseRewardFactor is used to calculate validator per-slot interest rate.
-	WhistleBlowingRewardQuotient uint64 `yaml:"WHISTLEBLOWING_REWARD_QUOTIENT"` // WhistleBlowingRewardQuotient is used to calculate whistler blower reward.
-	ProposerRewardQuotient       uint64 `yaml:"PROPOSER_REWARD_QUOTIENT"`       // ProposerRewardQuotient is used to calculate the reward for proposers.
-	InactivityPenaltyQuotient    uint64 `yaml:"INACTIVITY_PENALTY_QUOTIENT"`    // InactivityPenaltyQuotient is used to calculate the penalty for a validator that is offline.
-	MinSlashingPenaltyQuotient   uint64 `yaml:"MIN_SLASHING_PENALTY_QUOTIENT"`  // MinSlashingPenaltyQuotient is used to calculate the minimum penalty to prevent DoS attacks.
+	BaseRewardFactor            uint64 `yaml:"BASE_REWARD_FACTOR"`             // BaseRewardFactor is used to calculate validator per-slot interest rate.
+	WhistleBlowerRewardQuotient uint64 `yaml:"WHISTLEBLOWING_REWARD_QUOTIENT"` // WhistleBlowerRewardQuotient is used to calculate whistler blower reward.
```
Comment (prestonvanloon) [inline shared/params/config.go]: Do you need to rename WHISTLEBLOWING_REWARD_QUOTIENT ?

Comment (nisdas) [reply shared/params/config.go]: Alright, done

File: shared/mathutil/math_helper.go [hunk 2]
```diff
@@ -51,3 +51,21 @@ func ClosestPowerOf2(n uint64) uint64 {
 	exponent := math.Floor(math.Log2(float64(n)))
 	return PowerOf2(uint64(exponent))
 }
+
+// MaxOfTwoValues returns the larger integer of the two
+// given ones.
+func MaxOfTwoValues(a uint64, b uint64) uint64 {
```
Comment (prestonvanloon) [inline shared/mathutil/math_helper.go]: Can this just be `Max`?

File: shared/mathutil/math_helper.go [hunk 3]
```diff
@@ -51,3 +51,21 @@ func ClosestPowerOf2(n uint64) uint64 {
...
+	if a > b {
+		return a
+	}
+	return b
+}
+
+// MinOfTwoValues returns the smaller integer of the two
+// given ones.
+func MinOfTwoValues(a uint64, b uint64) uint64 {
```
Comment (prestonvanloon) [inline shared/mathutil/math_helper.go]: Can this just be `Min`?

File: shared/mathutil/math_helper.go (hunk 2 above)
Comment (prestonvanloon) [reply shared/mathutil/math_helper.go]: Please add commentary why this is better than math.Max, likewise for Min

**Pull request merged**